src/ripple/app/tests/AmendmentTable.test.cpp

```diff
@@ -678,25 +636,22 @@ class AmendmentTable_test final : public beast::unit_test::suite
             doRound (*table, i,
                 validators, votes, ourVotes, enabled, majority);
 
-            if (i < 14)
+            if (i < 13)
```

Comment:
Note: These numbers changed because we changed the way we calculate the threshold in `AmendmentTableImpl::doVoting` from rounding to floor.